code/modules/tips/tips.dm

```diff
@@ -0,0 +1,219 @@
+	category = "Admins"
+	weight = 3
+
+/datum/pomf_tip/antag_seeds
+	title = "Antag seeds"
+	desc = "There is no such thing as an antag seed. Stop asking."
+	category = "Admins"
+	weight = 1
+
+// -- Roleplay
+
+/datum/pomf_tip/coop
+	title = "Cooperation"
+	desc = "Cooperation is key. A friendly word or PDA message can make a much more fun round."
+	category = "Roleplay"
+	weight = 3
+
+/datum/pomf_tip/coop
+	title = "Losing is fun"
+	desc = "Sometimes things don't go your way. Try and enjoy the ride, there's always next shift and maybe your story helped make someone else's."
+	category = "Roleplay"
+	weight = 3
+
+/datum/pomf_tip/sniper
+	title = "Losing is fun"
+	desc = "Be polite. Be efficient. Have a plan to slip everyone you meet."
+	category = "Roleplay"
+	weight = 1
+
+/datum/pomf_tip/teaching
+	title = "Teach"
+	desc = "Teaching your fellow spaceman is a time honored tradition. Help the newbies out, pass on the torch."
+	category = "Roleplay"
+	weight = 3
+
+/datum/pomf_tip/forever
+	title = "There is never an end to ss13"
+	desc = "Don't forget. You're here forever."
+	category = "Roleplay"
+	weight = 1
+
+// -- General
+
+/datum/pomf_tip/ground_up
+	title = "Grinding items"
+	desc = "A lot of things can be ground up to make chemicals and materials. Try and grind up a Danitos package, if you dare!"
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/alt_click
+	title = "Containers"
+	desc = "You can alt-click containers to show their contents and put items in them without picking them up."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/hotkey_mode
+	title = "Hotkey mode"
+	desc = "You can press TAB to enable hotkey mode and use WASD movement, T for talk, and other keyboard shortcuts. Consult the wiki for a full list."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/pda_messages
+	title = "PDAs"
+	desc = "Your PDA has a lot of utility. Buy some programs for it, or send everyone in a department a message if they're not listening to the radio! Some can even be health scanners!"
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/antag_objectives
+	title = "Solo objectives"
+	desc = "If you lack inspiration or prefer to have a defined set of challenges, you can opt to have defined antagonist objectives by selecting the option in character setup."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/runechat
+	title = "Runechat"
+	desc = "In your options tab, 'runechat' refers to animations of the text said by players being displayed over their heads as they are talking."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/attack_anims
+	title = "Attack animations"
+	desc = "Confusing about who's hitting who? You can toggle attack animations in your preferences."
+	category = "General"
+	weight = 3
+
+// -- Wholesomefully contributed by xeroxemnas
+/datum/pomf_tip/trans_rights
+	title = "Trans rights!"
+	desc = "Trans rights are human rights. Trans lefts are also human lefts."
```

Comment:
okay you're funny but not really, delet it
```suggestion
	desc = "you will never be a woman"
```

Reply:
strange, i always interpreted that quote as transgenders not actually being human but we still need to treat them like so